src/lib/containers/entity/annotations/SchemaDrivenAnnotationEditor.tsx

```diff
@@ -197,7 +182,7 @@ export const SchemaDrivenAnnotationEditor = (
               </b>
             </Alert>
           )}
-          {entityJson && (!formData || isEmpty(formData)) && !schema && (
+          {entityJson && (!formData || isEmpty(formData)) && schema === null && (
```

Comment:
`!schema` may be true if there is no schema, as it is undefined. `schema === null` is only true when we know that there is no schema.